### DISCUSSION TITLE
provider that is not read will not be created?

### DISCUSSION THREAD

**Discussion by benny856694** [rrousselGit/riverpod#1902] [General]
The following provider is used to update title on windows platform, it seems it never get created, since no read is performed on it
```
final windowsTitleUpdateProvider = Provider((ref) {
  ref.listen(selectedFirmwareProvider, (previous, next) {
    var title = '${trans.appTitle.i18n}-[$next]';
    windowManager.setTitle(title);
  });
});
```

**Comment by rrousselGit** [comment]
Correct. Providers aren't created if you don't read them.

**Reply by benny856694** [reply]
So in this case, I use Provider Observer instead, any better way?
